File: 1791-richest-customer-wealth/1791-richest-customer-wealth.java
class Solution {
    public int maximumWealth(int[][] acc) {
       int max = Integer.MIN_VALUE;
       for(int[] row : acc) {
        int sum=0;
            for(int i : row){
                sum+= i;
            }
            max = Math.max(max,sum);
       }
       return max;
    }
}

File: 1791-richest-customer-wealth/1791-richest-customer-wealth.js
/**
 * @param {number[][]} accounts
 * @return {number}
 */
var maximumWealth = function(acc) {
    let max = -Infinity;
       for(let row of acc) {
        let sum=0;
            for(let i of row){
                sum+= i;
            }
            max = Math.max(max,sum);
       }
       return max;
};